x-pack/plugins/apm/server/routes/mobile/get_mobile_average_launch_time.ts

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import {
+  kqlQuery,
+  rangeQuery,
+  termQuery,
+} from '@kbn/observability-plugin/server';
+import { offsetPreviousPeriodCoordinates } from '../../../common/utils/offset_previous_period_coordinate';
+import { APP_LAUNCH_TIME, SERVICE_NAME } from '../../../common/es_fields/apm';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import { getOffsetInMs } from '../../../common/utils/get_offset_in_ms';
+import { APMEventClient } from '../../lib/helpers/create_es_client/create_apm_event_client';
+import { getBucketSize } from '../../../common/utils/get_bucket_size';
+import { Coordinate } from '../../../typings/timeseries';
+import { Maybe } from '../../../typings/common';
+
+export interface AvgLaunchTimeTimeseries {
+  currentPeriod: { timeseries: Coordinate[]; value: Maybe<number> };
+  previousPeriod: { timeseries: Coordinate[]; value: Maybe<number> };
+}
+
+interface Props {
+  apmEventClient: APMEventClient;
+  serviceName: string;
+  transactionName?: string;
+  environment: string;
+  start: number;
+  end: number;
+  kuery: string;
+  offset?: string;
+}
+
+async function getAvgLaunchTimeTimeseries({
+  apmEventClient,
+  serviceName,
+  transactionName,
+  environment,
+  start,
+  end,
+  kuery,
+  offset,
+}: Props) {
+  const { startWithOffset, endWithOffset } = getOffsetInMs({
+    start,
+    end,
+    offset,
+  });
+
+  const { intervalString } = getBucketSize({
+    start: startWithOffset,
+    end: endWithOffset,
+    minBucketSize: 60,
+  });
+
+  const aggs = {
+    launchTime: {
+      avg: { field: APP_LAUNCH_TIME },
```

Comment:
The avg aggregation might return long float number that could break the UI. Can we make sure the number is rounded to a specific number of decimal places, such as 2 or 1.

Reply:
Thank you! I didn't think about this, I'll make the changes 👍

Reply:
It's done now, I've made the changes [here](https://github.com/elastic/kibana/pull/170773/files#diff-8e2c738e19199614fe5fdc76e6f63e53eaacf2c305f1c69cfcca63a10f51d867R132) before displaying the data. Thanks!